src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -83,9 +95,67 @@
     }
 @end
 
+@private clientStreamingMethodSampleCode(apiMethod)
+    $requests = [$request];
+    
+    // Write data to server and wait for a response
+    $streamingResponse = {@methodCallSampleCode(apiMethod)};
+    $result = $streamingResponse->writeAllAndWait($requests);
```

Comment:
What do you think about `$clientStream`, `$serverStream` and `$bidiStream`? I am not fully convinced that is right though, as they aren't really streams... maybe `$clientStreamConnection`? `$clientStreamingConnection`?